### PR TITLE
Handle google-cloud restructure for auto_prepend_file path

### LINKED ISSUE #427: Handle new file structure for google-cloud-php for Stackdriver Integration
The structure of the google-cloud-php repository will change in the next version. We rely on a path to a prepend file when `enable_stackdriver_integration: true` is configured in the user's `app.yaml`

### PR DESCRIPTION
**Pull request by chingor13**
Fixes #427

### CODE REVIEW

File: php-base/stackdriver-files/locate_stackdriver_prepend.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+use Google\Cloud\ErrorReporting\Bootstrap;
+
+if (class_exists(Bootstrap::class)) {
+    $reflection = new \ReflectionClass(Bootstrap::class);
+    if ($reflection->hasMethod('prependFileLocation')) {
+        echo Bootstrap::prependFileLocation() . PHP_EOL;
+    } else {
+        // default to same directory as the Bootstrap.php
+        echo dirname(Bootstrap::getFileName()) . DIRECTORY_SEPARATOR . 'bootstrap.php' . PHP_EOL;
```
Comment (tmatsuo) [inline php-base/stackdriver-files/locate_stackdriver_prepend.php]: Should we use realpath to make it an absolute path?

Comment (chingor13) [reply php-base/stackdriver-files/locate_stackdriver_prepend.php]: Sure, will do.

File: php-base/stackdriver-files/locate_stackdriver_prepend.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+use Google\Cloud\ErrorReporting\Bootstrap;
+
+if (class_exists(Bootstrap::class)) {
+    $reflection = new \ReflectionClass(Bootstrap::class);
+    if ($reflection->hasMethod('prependFileLocation')) {
+        echo Bootstrap::prependFileLocation() . PHP_EOL;
+    } else {
+        // default to same directory as the Bootstrap.php
+        echo realpath(dirname($reflection->getFileName()) . '/bootstrap.php') . PHP_EOL;
```
Comment (tmatsuo) [inline php-base/stackdriver-files/locate_stackdriver_prepend.php]: '/prepend.php'?

Comment (chingor13) [reply php-base/stackdriver-files/locate_stackdriver_prepend.php]: Whoops, yes. We really need to test this.

**Comment by chingor13** [comment]
@tmatsuo Build is fixed. PTAL

**Pull request merged**